src/components/SchemaResourcePage/resourceTypes.tsx

```diff
@@ -39,13 +40,21 @@ export type SchemaResourceType = {
     default?: any;
     hidden?: boolean;
   }[];
+  analyticsEventName: {
```

Comment:
@moshloop Because of how the UI is setup, I found it easier to embedded a resource events to the resources list, so when you delete, create or update, we are able to send the appropriate event.